sdk/eventhub/azure-eventhubs/azure/eventhub/_connection_manager.py

```diff
@@ -74,4 +80,7 @@ def reset_connection_if_broken(self):
 
 
 def get_connection_manager(**kwargs):
+    connection_mode = kwargs.get("connection_mode", _ConnectionMode.SeparateConnection)
```

Comment:
we have not yet finished share connection implementation, so for this preview it won't be public.
But once we've done the amqp share connection and related stuff in next release, it would be made public